packages/@coorpacademy-components/src/molecule/title-radio-wrapper/test/fixtures/select-cohortes.ts

```diff
@@ -0,0 +1,46 @@
+export default {
+  props: {
+    radioWithTitle: {
+      title: {
+        type: 'form-group',
+        title: 'Select one or several cohortes',
+        subtitle: 'in the drop-down list'
+      }
+    },
+    field: {
+      theme: 'coorpmanager',
+      placeholder: 'Select cohorte(s)',
+      title: 'Cohortes',
+      description: 'You should select a cohorte',
+      multiple: true,
+      disabled: false,
+      error: '',
+      hint: '',
+      options: [
+        {
+          name: 'ABRERA FCM',
+          value: 'abrera-fcm',
+          selected: true
+        },
+        {
+          name: 'ABRERA FIS',
+          value: 'abrera-fis',
+          selected: false
+        },
+        {
+          name: 'ALLARIZ',
+          value: 'allariz',
+          selected: false
+        },
+        {
+          name: 'ALMUSSAFESS FCM',
+          value: 'almussafes-fcm',
+          selected: false
+        }
+      ],
+      onChange: (value: any) => console.log('onChange', value),
+      onError: () => true
+    },
+    childType: 'selectMultiple'
```

Comment:
The value `selectMultiple` should be in kebab case or in camel case ? (The other type is in kebab case `drag-and-drop`). It should be standardized.

Reply:
@EmelineLeduc  you're right, it should be standardized. I commited the changes